si_ps_conv_fastcorrsApp/Db/kick_conv.template

```diff
@@ -0,0 +1,29 @@
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):$(CURRSUFFIX)") {
+    field(PINI, "YES")
```

Comment:
I don't think it makes a lot of sense to use `PINI=YES` in a record where the input record is `CP`. That means it will be processed immediately when the PV connects, and processing before that will use a zeroed out value.